octree.h

```diff
@@ -2722,10 +2742,10 @@ namespace OrthoTree
     }
 
     // Create
-    template<typename TExecutionPolicy = std::execution::unsequenced_policy>
+    template<typename TExecutionPolicy = std::execution::unsequenced_policy, AccessorConcept TAccessor = std::span<const TBox>>
     static void Create(
       OrthoTreeBoundingBox& tree,
-      std::span<TBox const> const& boxes,
+      TAccessor const& boxes,
```

Comment:
You also have to rewrite the `entityID` logic in the `std::for_each`-es. (Same issue as above.)